dwl.c

```diff
@@ -2679,8 +2679,6 @@ zoom(const Arg *arg)
 	 * front */
 	if (!sel)
 		sel = c;
-	wl_list_remove(&sel->link);
-	wl_list_insert(&clients, &sel->link);
 
 	focusclient(sel, 1);
```

Comment:
It looks like the focusclient here causes some weird behavior when closing X11 windows (i.e. the next window in the fstack is not necessarily selected).